examples/lib/stories/input/keyboard.dart

```diff
@@ -1,13 +1,12 @@
 import 'dart:ui';
 
 import 'package:flame/game.dart';
-
 import 'package:flame/input.dart';
 import 'package:flame/palette.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 
-class KeyboardGame extends Game with KeyboardEvents {
+class KeyboardGame extends FlameGame with KeyboardEvents {
```

Comment:
would this be a good candiate for a Game example? since it was before & it uses no components

Reply:
Done